subcommands/targets/offline-update.go

```diff
@@ -286,16 +287,19 @@ func downloadTufRepo(factory string, target string, tag string, prod bool, wave
 		}
 		ver += 1
 	}
-
-	meta, err := api.TufTargetMetadataRefresh(factory, target, tag, expiresIn, prod, wave)
+	bundleTargetsPointer, err := getBundleTargetsMeta(dstDir)
```

Comment:
I like the current name better :). A reference and pointer are kind of synonyms in this context.

Reply:
I named this variable with `pointer` prefix because in there was another variable with `bundleTargets` in this code scope. However, after refactoring there is no such variable anymore, so the `pointer` one can be just `bundleTargets` indeed.

Reply:
Fixed in the forced push.